docs/content/components/list.mdx

```diff
@@ -0,0 +1,127 @@
+---
```

Comment:
All the `className` attributes need to be fixed below.

Reply:
<img width="939" alt="Screen Shot 2021-07-29 at 3 36 13 PM" src="https://user-images.githubusercontent.com/85901679/127562255-052fc599-be16-4456-b2f9-887b7c484077.png">

This is why they were set to className instead of class.

Reply:
https://github.com/NCIOCPL/ncids/pull/203/checks?check_run_id=3195981752

Reply:
Oh I see you have found a bug. Well className is not valid HTML. So this seems like a different issue we need to log. grr. I see the button example has that incorrectly in its HTML.

Reply:
Ah, I believe there is a small miscommunication here.  The display html should have class, not the executed block.  
<img width="753" alt="Screen Shot 2021-08-03 at 11 24 56 AM" src="https://user-images.githubusercontent.com/45469809/128042541-e70b4211-7739-46e5-aec1-f760f3ba628e.png">